src/node/historical_queries_adapter.cpp

```diff
@@ -10,6 +10,58 @@
 #include "node/rpc/network_identity_subsystem.h"
 #include "node/tx_receipt_impl.h"
 
+#include <t_cose/t_cose_sign1_sign.h>
+
+namespace
+{
+  static constexpr int64_t LEAF_LABEL = 404; // TBD
+  static constexpr int64_t PATH_LABEL = 404 + 1; // TBD
+
+  std::vector<uint8_t> from_string(std::string_view s)
```

Comment:
suggest we move that to ds, it's often useful

Reply:
It's actually hasn't been used in this PR anymore, so I deleted it..